### PR TITLE
Added report types to readme

### PR DESCRIPTION
**Pull request by 4uto**
stolen from "istanbul help reports"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,10 +157,44 @@ To view your coverage report:
...
 <img width="500" src="screen.png">
 
-you can use any reporters that are supported by istanbul:
+you can use any reporters that are supported by istanbul and even use several at once:
 
 ```bash
-nyc report --reporter=lcov
+nyc report --reporter=<format> --reporter=<another format>
+```
+<format> is one of:
```
Comment (addaleax) [inline README.md]: This `format` should probably be escaped here, e.g. `` `format` ``? Otherwise it doesn’t display in the rendered README. :)

**Comment by 4uto** [comment]
I escaped a <format>, thanks Anna Henningsen

**Comment by bcoe** [comment]
@4uto @addaleax sorry to be _that nit-picky OSS maintainer_, but I think I would rather see this new content in this README.md:

https://github.com/istanbuljs/istanbul-reports

With a section added to the `nyc` README that says, `see a full list of reports here`.

I think this would make it more likely that we keep documentation up to date as we add additional reports. What do you think?

**Comment by bcoe** [comment]
@4uto just checking in, are you alright with this change?

**Comment by 4uto** [comment]
@bcoe Sorry for the delay. My new baby is keeping me away from the computer.

Your thinking is sound. I was just under the impression that nyc was the new black and that istanbul wasn't "in the front" anymore. I'll go make the change over there and then come back here with an update like you suggested.

**Comment by bcoe** [comment]
@4uto thanks, and babies are more important that JavaScript test coverage! 👶 

My personal goal, is to keep the individual READMEs fairly concise, but to gradually flesh out http://istanbul.js.org/ further:

https://github.com/istanbuljs/istanbuljs.github.io

**Pull request closed without merging**